es-md/docs/index.html

```diff
@@ -29,7 +29,7 @@ <h2 class="subtitle">Cómo crear y dar lecciones que funcionen y construir una c
 <p>La traducción de este libro al español está dedicada a la memoria de
 <a href="https://es.wikipedia.org/wiki/Rebeca_Guber">Rebeca Cherep de Guber</a>.</p>
 <p>Todas las regalías de la venta de este libro (versión en español) se donan a<br/>
-<a href="https://metadocencia.org">MetaDocencia</a>,<br/>
+[MetaDocencia][k0015],<br/>
```

Comment:
```suggestion
[MetaDocencia][metadocencia],<br/>
```

Reply:
You use this key at [es-md/community/index.md](https://github.com/gvwilson/teachtogether.tech/pull/475/files#diff-3f12575637f7ffc5aaab0da737a68a074b57156b56cb8ebdbbbe565a334ed244)

Reply:
Fixed in f488686